src/GeoJSON.Text.Test.Benchmark/Deserialize/DeserializeFeatureLinestring.cs

```diff
@@ -0,0 +1,29 @@
+﻿using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Jobs;
+
+namespace GeoJSON.Text.Test.Benchmark.Deserialize
+{
+    [SimpleJob(RuntimeMoniker.Net60, baseline: true)]
+    [SimpleJob(RuntimeMoniker.Net50)]
+    [SimpleJob(RuntimeMoniker.NetCoreApp31)]
+    [MemoryDiagnoser]
+    public class DeserializeFeatureLinestring
+    {
+        string fileContents = "";
+
+        [GlobalSetup]
+        public void Setup()
+        {
+            fileContents = JsonEmbeddedFileReader.GetExpectedJson("DeserializeFeatureLinestring");
+        }
+
+#pragma warning disable CS8603 // Possible null reference return.
```

Comment:
If we're going to supress the null checks for this project would it be easier to set at project level? `<nullable>disable</nullable>`

Reply:
Now throws NullReferenceException if result is null.